src/components/Card.js

```diff
@@ -27,7 +27,7 @@ export default class Card {
 
     this.cardIsLiked = this.cardIsLiked.bind(this);
   }
-  /* 
```

Comment:
**Might be better**

`this._element.querySelector(".card__like-counter")` and other elements




All elements that are found every time you call a method should be defined as **class fields** in the constructor, `_setEventListeners` (or in `generateCard`) only **1 time** not to waste resources on searching them again and again.

`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.
There can be `this._inputs`, `this._submitButton`, `this._cardImage`, `this._form` and so on.